### PR TITLE
Sidebar menu rethink

### PR DESCRIPTION
**Pull request by vargaat**
refs: MBL-15270, MBL-15275, MBL-15276
affects: Parent, Student, Teacher
release note: Updated menu design.

test plan: Enjoy the new menu, everything should work just as before.

### CODE REVIEW

File: Core/Core/Profile/SideMenu/SideMenuFooterView.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+struct SideMenuFooterView: View {
+    var body: some View {
+        if let version = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String {
+            HStack {
+                Text("v. \(version)").padding(.leading, 10).font(.regular14).foregroundColor(.ash)
```
Comment (vargaat) [inline Core/Core/Profile/SideMenu/SideMenuFooterView.swift:25]: We should use `Text(verbatim:` here, otherwise this will be translated.

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>



<details><summary>:x: XCTest failed: CoreTests/CacheManagerTests/testClearNoNeeded</summary>


    Assertion Failure at CacheManagerTests.swift:71: XCTAssertTrue failed
</details>


[MBL-15270](https://instructure.atlassian.net/browse/MBL-15270)
[MBL-15275](https://instructure.atlassian.net/browse/MBL-15275)
[MBL-15276](https://instructure.atlassian.net/browse/MBL-15276)

#### Affected Apps: Parent, Student, Teacher

#### Release Note: 
Updated menu design
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/73102714/p/741ca47267b0ff16091fb66e43c0749d">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/73102720/p/1d667187d135548e2b2fb1d85f92df3f">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/73102707/p/b0ebf3b22d3bd71f28b722669800b8d4">
    
  </a>
</details>

File: Core/Core/Profile/SideMenu/SideMenuFooterView.swift (hunk 1 above)
Comment (idevainstr) [reply Core/Core/Profile/SideMenu/SideMenuFooterView.swift]: fixed.

**Review by balintbartok — DISMISSED** [review]
QA + 1

**Comment by vargaat** [comment]
QA+1

**Review by balintbartok — APPROVED** [review]

**Pull request merged**